R/read_stuff.R

```diff
@@ -11,62 +11,65 @@
 #' @examples
 #' some_path <- "path to your file"
 #' read_stuff(some_path)
-
 read_stuff <- function(x) {
   # check inputs
   checkmate::assertCharacter(x, len = 1L)
 
-  
   # do stuff
   x
 }
 
 
 
-
-
-
-#' function to read data from all supported file types in the `rio` package. 
+#' function to read data from all supported file types in the `rio` package.
 #' @param file.path the path to the file to be read. When several files need to be imported from a directory, this should be the path to that directory
-#' @param sep the separator between the columns in the file. This is only required for space-separated files. 
+#' @param sep the separator between the columns in the file. This is only required for space-separated files
 #' @param format a string used to specify the file format. This is useful when a file does not have an extension, or has a file extension that does not match its actual type
 #' @param which a string used to specify the name of the excel sheet to import
 #' @param pattern when specified, only files with this suffix will be imported from the specified directory
 #' @returns  a data frame (import from file) or a list (import several files from directory)
-#' @examples data = read_from_file(file.path = "/path/to/test.txt", sep=NULL, format=NULL, which=NULL, pattern=NULL)
+#' @examples data <- read_from_file(file.path = system.file("extdata", "test.txt", package = "readepi"), sep = NULL, format = NULL, which = NULL, pattern = NULL)
```

Comment:
```suggestion
#' @examples 
#' \dontrun{
#' data <- read_from_file(
#'    file.path = system.file("extdata", "test.txt", package = "readepi"), 
#'    sep = NULL, 
#'    format = NULL, 
#'    which = NULL, 
#'    pattern = NULL
#' )
#' }
```

Reply:
Not sure this is the problematic example that needs the credentials but you can use this as an example to fix it where required.